docs/casts/simple_provenance_tracking.sh

```diff
@@ -39,9 +39,11 @@ run "datalad rerun"
 run "git log --oneline --graph --name-only @~3..@"
 
 say "In this case, a new commit isn't created because the output file didn't change. But let's say we add a step that displaces the Lily's pixels by a random amount."
-run "datalad run convert -spread 10 st-bernard.jpg st-bernard-displaced.jpg"
+run "export SEED=38"
+run "datalad run 'convert -spread 10 -seed \$SEED st-bernard.jpg st-bernard-displaced.jpg'"
 
-say "Now, if we rerun the previous command, a new commit is created because the output's content changed."
+say "Notice that we're setting the random seed outside of the run command. Now, if we changed the seed and rerun the previous command, a new commit is created because the output's content changed."
+run "export SEED=83"
```

Comment:
cute -- will be even more realistic demonstration for "irreproducibility" since environment does matter